src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,141 @@
+                        "validationErrors": {
+                        "$.codeValue": [
+                            "CodeValue is required and should not be left empty."
+                        ]
+                  }
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data Forbidden
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "",
+                        "description": "Wrong Value",
+                        "namespace": "uri://.org/wrong",
+                        "shortDescription": "Wrong Value"
+                    }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                        "$.codeValue": [
+                                "CodeValue is required and should not be left empty."
+                            ]
+                        }
+                    }
+                  """
+        @ignore
+        Scenario: Verify error handling with POST using empty body
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                    }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                            "$.codeValue": [
+                                "CodeValue is required."
+                                ],
+                            "$.namespace": [
+                                "Namespace is required."
+                                ],
+                            "$.shortDescription": [
+                                "ShortDescription is required."
+                                ]
+                        }
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify POST of existing record without changes
+                          #an existing record to
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
```

Comment:
Perhaps it would make sense to duplicate the "when" statement as a "Given" statement, so that the POST occurs twice.

Reply:
Done